Dockerfile

```diff
@@ -80,6 +78,9 @@ RUN cget -p $PREFIX install ccache@v4.1
 # Install newer cmake for onnx runtime
 RUN cget -p /opt/cmake install kitware/cmake@v3.13.0
 
+# Install onnx 1.8.1
+RUN pip3 install onnx numpy typing pytest
```

Comment:
Isn't this already installed in the `install_prereqs.sh` script?

Reply:
Also, the pinned versions should not be dropped.

Reply:
Yes, you are right. I removed this line and the line for 1.7.0 in the Dockerfile. 

Actually, I am wondering whether we can install two python versions (like 3.6 and 3.7), and install different onnx versions on them?

Reply:
Is the reason to install two different python version so we can have two different onnx versions installed? As we can create two different venvs to install onnx. I dont think we should do that in this PR, but we could look into cmake automatically setting up the venvs so the the tests will know where to find them.

Reply:
yes, we want two different onnx versions installed, so we can test both onnx versions.
OK, we can push later to add that.